### PR TITLE
Enable GitHub Actions CI

### PR DESCRIPTION
**Pull request by e0ne**
This patch introduces following CI jobs:
* lint, build and unit-test run
* publish image to GHCR on each merge
* publish image to GHCR on each release

### CODE REVIEW

**Comment by e0ne** [comment]
Since GitHub doesn't run actions from forks, here is example of the builds for the same commit: https://github.com/e0ne/ib-sriov-cni/pull/1

File: .github/workflows/buildtest.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    push:
+    pull_request:
+    schedule:
+      - cron: "0 8 * * 0" # every sunday
+jobs:
+  build:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x, 1.15.x, 1.16.x]
+        os: [ubuntu-latest]
```
Comment (moshe010) [inline .github/workflows/buildtest.yml:13]: ubuntu 20.04?

File: .github/workflows/image-push-master.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+name: "Push images on merge to master"
+on:
+  push:
+    branches:
+      - master
+jobs:
+  build-and-push-image-sriov-cni:
```
Comment (moshe010) [inline .github/workflows/image-push-master.yml:7]: ib-sriov-cni

File: .github/workflows/image-push-release.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+name: "Push images on release"
+on:
+  push:
+    tags:
+      - v*
+jobs:
+  build-and-push-image-sriov-cni:
```
Comment (moshe010) [inline .github/workflows/image-push-release.yml:7]: ib-sriov-cni

File: .github/workflows/image-push-release.yml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: ghcr.io/${{ github.repository }}
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
```
Comment (moshe010) [inline .github/workflows/image-push-release.yml:15]: are we using buildx to support multi arch?  if yes were that arch defined?

**Review by moshe010 — CHANGES_REQUESTED** [review]

File: .github/workflows/image-push-release.yml (hunk 4 above)
Comment (e0ne) [reply .github/workflows/image-push-release.yml:15]: We don't use it right now. It's done as same as for sriov-cni and other k8snetworkplumbingwg's projects to have the same approach over the whole organization

**Review by moshe010 — APPROVED** [review]

**Pull request merged**